server/cp/output/formatter/jimi.py

```diff
@@ -642,6 +642,12 @@ def _format_content(self, item, is_broadcast):
                 elem.tag = "strong"
             elif elem.tag == "i":
                 elem.tag = "em"
+
+            # Remove whitespace and empty tags
+            if elem.text is not None and not elem.text.strip():
+                elem.text = None
+                etree.strip_tags(tree, elem.tag)
```

Comment:
imo you should call it on `elem` not `tree` otherwise this will remove the tag from the whole body, try to add another `<b>` with some content to the test case